AlfrescoApp/Constants/AnalyticsConstants.h

```diff
@@ -0,0 +1,192 @@
+extern NSString * const kAnalyticsEventActionOpen;
+extern NSString * const kAnalyticsEventActionSync; // 2.3
+extern NSString * const kAnalyticsEventActionEmail;
+extern NSString * const kAnalyticsEventActionEmailLink;
+extern NSString * const kAnalyticsEventActionSendForReview;
+extern NSString * const kAnalyticsEventActionComment;
+extern NSString * const kAnalyticsEventActionFavorite;
+extern NSString * const kAnalyticsEventActionUnfavorite;
+extern NSString * const kAnalyticsEventActionLike;
+extern NSString * const kAnalyticsEventActionUnlike;
+extern NSString * const kAnalyticsEventActionPrint;
+extern NSString * const kAnalyticsEventActionCall;
+extern NSString * const kAnalyticsEventActionSkype;
+extern NSString * const kAnalyticsEventActionShowInMaps;
+extern NSString * const kAnalyticsEventActionMembership;
+extern NSString * const kAnalyticsEventActionReassign;
+extern NSString * const kAnalyticsEventActionComplete;
+extern NSString * const kAnalyticsEventActionRunSimple;
+extern NSString * const kAnalyticsEventActionRun;
+extern NSString * const kAnalyticsEventActionAnalytics;
+extern NSString * const kAnalyticsEventActionClearData;
+
+// Labels
+extern NSString * const kAnalyticsEventLabelOnPremise;
+extern NSString * const kAnalyticsEventLabelCloud;
+extern NSString * const kAnalyticsEventLabelNetwork;
+extern NSString * const kAnalyticsEventLabelProfile;
+extern NSString * const kAnalyticsEventLabelDocumentMimetype;
+extern NSString * const kAnalyticsEventLabelFolder;
+extern NSString * const kAnalyticsEventLabelTakePhotoOrVideo;
+extern NSString * const kAnalyticsEventLabelRecordAudio;
+extern NSString * const kAnalyticsEventLabelPhone;
+extern NSString * const kAnalyticsEventLabelMobile;
+extern NSString * const kAnalyticsEventLabelEnterprise;
+extern NSString * const kAnalyticsEventLabelSMS;
+extern NSString * const kAnalyticsEventLabelChat;
+extern NSString * const kAnalyticsEventLabelCall;
+extern NSString * const kAnalyticsEventLabelVideoCall;
+extern NSString * const kAnalyticsEventLabelUser;
+extern NSString * const kAnalyticsEventLabelCompany;
+extern NSString * const kAnalyticsEventLabelJoin;
+extern NSString * const kAnalyticsEventLabelLeave;
+extern NSString * const kAnalyticsEventLabelCancel;
+extern NSString * const kAnalyticsEventLabelEnable;
+extern NSString * const kAnalyticsEventLabelDisable;
+extern NSString * const kAnalyticsEventLabelFiles;
+extern NSString * const kAnalyticsEventLabelFolders;
+extern NSString * const kAnalyticsEventLabelSites;
+extern NSString * const kAnalyticsEventLabelPeople;
+extern NSString * const kAnalyticsEventLabelSyncedFolders;
+extern NSString * const kAnalyticsEventLabelSyncedFiles;
+extern NSString * const kAnalyticsEventLabelDisableConfig;
+extern NSString * const kAnalyticsEventLabelPartial;
+extern NSString * const kAnalyticsEventLabelFull;
+
+#pragma mark - Custom Metrics
+
+typedef NS_ENUM(NSUInteger, AnalyticsMetric)
+{
+    AnalyticsMetricNone                = 0,
```

Comment:
Good that you've added the explicit values here, but could you also add a warning comment that these values *must* match the index values in the Google Analytics config please?